pkg/executables/helm.go

```diff
@@ -126,7 +138,10 @@ func (h *Helm) InstallChartFromName(ctx context.Context, ociURI, kubeConfig, nam
 // InstallChart installs a helm chart to the target cluster.
 func (h *Helm) InstallChart(ctx context.Context, chart, ociURI, version, kubeconfigFilePath, namespace, valueFilePath string, values []string) error {
 	valueArgs := GetHelmValueArgs(values)
-	params := []string{"install", chart, ociURI, "--version", version}
+	params := []string{"upgrade", "--install", chart, ociURI, "--version", version}
```

Comment:
for my own understanding, what's the difference between install and upgrade?
This is going to be a change for all consumers of this method.

Reply:
`upgrade --install` is an "upsert" of sorts. If the release doesn't exist, it's created. If the release exists, it's upgraded. That is to say, it attempts to be idempotent.

`install` will fail if the release already exists, which doesn't work well with a reconcile situation.